examples/get_realtime_data.py

```diff
@@ -2,9 +2,15 @@
 import time
 import logging
 
-r = urx.Robot("192.168.111.134", use_rt=True, urFirm=5.1)
 
 if __name__ == "__main__":
+    while 1:
+        try:
+            r = urx.Robot("192.168.137.129", use_rt=True, urFirm=5.1)
+            break
+        except:
+            pass
+
```

Comment:
example code must be as simple as possible the lines over are useless complexity.
Looks like you have a network issue. That is the one that should be fixed, not every library on earth